totalRP3/Modules/Register/Characters/RegisterMisc.lua

```diff
@@ -167,15 +170,18 @@ local function displayRPStyle(context)
 			frame.FieldName:SetText(fieldData.name);
 			local dropDown = frame.Values;
 			local readOnlyValue = frame.FieldValue;
+			local tooltipFrame;
 			if context.isPlayer then
 				frame:SetHeight(30);
 				dropDown:SetSelectedValue(selectedValue);
 				dropDown:Show();
 				readOnlyValue:Hide();
+				tooltipFrame = dropDown;
 			else
 				frame:SetHeight(24);
 				dropDown:Hide();
 				readOnlyValue:Show();
+				tooltipFrame = readOnlyValue;
```

Comment:
Need to have a think on this; the general behaviour we've been trending toward is to make it so that hovering over values shows a tooltip if the configured thing is truncated.

Given these dropdowns have known fixed values I _expect_ all of them fit within the frame just fine and will never truncate, but should probably test just to be sure.

Reply:
Checked on every locale we offer, there seems to be no truncating on the readOnlyValue field.